commands/operator-sdk/cmd/scorecard/resource_handler.go

```diff
@@ -43,6 +43,27 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
+type cleanupFn func() error
+
+// waitUntilReady waits until the status block of the CR currently being tested exists. If the timeout
+// is reached, it simply continues and assumes there is no status block
+func waitUntilReady(obj *unstructured.Unstructured) error {
```

Comment:
Perhaps rename this to be a bit more descriptive of what the wait is for.
Something like `waitUntilCRStatusExists(cr *unstructured.Unstructured)`.